peer/brontide.go

```diff
@@ -731,6 +731,8 @@ func NewBrontide(cfg Config) *Brontide {
 		)
 	}
 
+	peerCtx, _ := p.cg.Create(context.Background())
+
 	p.pingManager = NewPingManager(&PingManagerConfig{
```

Comment:
ideally we dont pass contexts via constructors but instead explicitly via "Start" calls. ctxs should only be passed to a constructor if the actual constructor makes calls that require a ctx.